compliance_checker/cf/cf.py

```diff
@@ -122,7 +123,13 @@ def _find_cf_standard_name_table(self, ds):
         # Try to parse this attribute to get version
         version = None
         if 'cf standard name table' in standard_name_vocabulary.lower():
-            version = standard_name_vocabulary.split()[-1]
+            version = [s.strip('(').strip(')').strip('v').strip(',') for s in standard_name_vocabulary.split()]
+            # This assumes that table version number won't start with 0.
+            version = [s for s in version if s.isdigit() and len(s) <=2 and not s.startswith('0')]
```

Comment:
This is ugly and assumes that the number we are looking for ([1-42 at the moment](http://cfconventions.org/standard-names.html)) won't start with `0` and is not longer than two digits. Still, we can find ambiguous digits for the version and I'd rather escape to the packaged version.

I am open to suggestions but I would prefer a more "parseable" standard for that property.